assets/app/scripts/filters/resources.js

```diff
@@ -781,6 +781,12 @@ angular.module('openshiftConsole')
   })
   .filter('humanizeResourceType', function() {
     return function(resourceType) {
+      if (!resourceType) {
+        return resourceType;
+      }
+
+      // Handle kinds like "ReplicationController" as well
+      resourceType = resourceType.toLowerCase();
```

Comment:
The map treats "replicationcontroller" and "replicationcontrollers" differently, so that will break existing callers. We need to clean it up when we fix the other kind/resource bug.